subprojects/docs/src/snippets/bestPractices/logicDuringConfiguration-do/groovy/build.gradle

```diff
@@ -12,8 +12,9 @@ dependencies {
 }
 
 tasks.register('printArtifactNames') {
+    FileCollection compileClasspath = configurations.compileClasspath
```

Comment:
This one is exposed at https://docs.gradle.org/current/userguide/authoring_maintainable_build_scripts.html#sec:minimize_logic_executed_configuration_phase, maybe docs should reflect the new sample structure.

Reply:
What do you mean by new sample structure, Mike?

Reply:
basically, now we have to cache the configuration into a variable at the configuration time. It might be worth highlighting that this doesn't resolve the configuration at the configuration time. Especially for Kotlin, where it is actually a `Provider<Configuration>`, and we call `get` on it. Well, for the configuration cache the Configuration is resolved before the execution time, but this is an implementation detail :)